pkg/validations/validations.go

```diff
@@ -197,3 +200,11 @@ func ValidateCaCertificate(certificate string) error {
 
 	return nil
 }
+
+// RFC 1123 (https://datatracker.ietf.org/doc/html/rfc1123#page-13)
+// states that domains cannot resemble the format ##.##.##.##
+func isDottedDecimalDomain(domain string) bool {
+	regex := `([\d]+\.){3}[\d]+`
+	return regexp.MustCompile(regex).MatchString(domain)
+
```

Comment:
Bonus line not needed.

Reply:
Thanks for catching that! Removed the extra line